In [7]:
#df = read.table("/shared/3/projects/benlitterer/localNews/MVP1/10articleTopics.csv",sep=",", allowEscapes=TRUE)
#df = read.csv("/shared/3/projects/benlitterer/localNews/MVP1/articleTopicsQuarter.csv")
library(readr)
library(lme4)
library(lubridate)
library(dplyr)

## want to work on aggregate data now... 

In [11]:
df = read_csv("/shared/3/projects/benlitterer/localNews/MVP1/outletMeans.csv")

New names:
• `` -> `...1`
Rows: 310 Columns: 88
── Column specification ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Delimiter: ","
chr  (1): source
dbl (87): ...1, Trump, Crime/Prosecution, Subscription_Service, Media, Labor...

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


In [14]:
colnames(df)

[1] "...1"                               "source"                            
 [3] "Trump"                              "Crime/Prosecution"                 
 [5] "Subscription_Service"               "Media"                             
 [7] "Labor/Pandemic/Immigration"         "Isreal/Palestine/India"            
 [9] "School/Universities"                "Royal_Family/Stars"                
[11] "NS1"                                "Outdoor Sports"                    
[13] "Business"                           "Data/Information/Privacy"          
[15] "Coronavirus_General"                "States1"                           
[17] "Congress"                           "Middle_East_Conflict"              
[19] "NS2"                                "Community"                         
[21] "Military"                           "Democratic_Primary"                
[23] "NS3"                                "NS4"                               
[25] "China"                              "BLM/George_Floyd"                  
[27] "Sports1"                            "Police_Activity"                   
[29] "Months"                             "Trump/Scandals"                    
[31] "Southern/Midwestern/Eastern_States" "Southwestern_States"               
[33] "Severe_Weather"                     "Travel/Flights"                    
[35] "Baseball"                           "Religion"                          
[37] "Basketball"                         "Race"                              
[39] "Construction"                       "Europe"                            
[41] "Political_Scandal_General"          "2020_Election"                     
[43] "Veterans"                           "America_as_Institution"            
[45] "Hockey"                             "Family"                            
[47] "German_Words"                       "Art/Books/Museums"                 
[49] "Football"                           "Food Service"                      
[51] "NS5"                                "Supreme_Court"                     
[53] "Covid_Illness"                      "Sports2"                           
[55] "Climate_Fossil_Fuel"                "Death/Funeral"                     
[57] "Covid_Economic"                     "Vaccination"                       
[59] "Pandemic_Funding"                   "Covid_Hospitals_Harm_Mitigation"   
[61] "Covid_Negatives/Deaths"             "Britain"                           
[63] "NS6"                                "Entertainment"                     
[65] "Health_Products"                    "Science/Space"                     
[67] "Wildlife_Farms"                     "NS7"                               
[69] "City_Government"                    "Soccer"                            
[71] "2020_Election_Fradulence"           "Community_Public_Service"          
[73] "total_population"                   "white_pct"                         
[75] "black_pct"                          "hispanic_pct"                      
[77] "nonwhite_pct"                       "foreignborn_pct"                   
[79] "female_pct"                         "age29andunder_pct"                 
[81] "age65andolder_pct"                  "median_hh_inc"                     
[83] "clf_unemploy_pct"                   "lesshs_pct"                        
[85] "lesscollege_pct"                    "lesshs_whites_pct"                 
[87] "lesscollege_whites_pct"             "rural_pct"

In [24]:
grep( "Death/Funeral", colnames(df))
colnames(df)[56] = "Death_Funeral"

[1] 56

In [46]:
mean(df$Race*100)

[1] 0.8361144

# Interpretation: Race ~ lesscollege_pct
- explains 14.5% of variance (R^2) 
- an increase in 1 percent of people who have less than a college degree in an outlet's county is associated with a .0234 percent decrease in articles about race. (highly significant p-value)  
- Race makes up .836% of the mean on average 

In [32]:
rural_lm = lm(Race*100~lesscollege_pct, data=df)
summary(rural_lm)


Call:
lm(formula = Race * 100 ~ lesscollege_pct, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-0.9903 -0.3921 -0.1704  0.1357  6.0387 

Coefficients:
                 Estimate Std. Error t value Pr(>|t|)    
(Intercept)      2.496491   0.231040  10.805  < 2e-16 ***
lesscollege_pct -0.023391   0.003202  -7.305 2.39e-12 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.7297 on 308 degrees of freedom
Multiple R-squared:  0.1477,	Adjusted R-squared:  0.1449 
F-statistic: 53.36 on 1 and 308 DF,  p-value: 2.39e-12


## Interpretation: Death_Funeral  ~ rural_pct
- An increase in 1 percent rurality in a county is associated with an increase of .13 percent in the Death_Funderal topic
- So an area with 70% rurality is inferred to have an increase of ~4% of Death in its local news coverage 
- Highly significant p-value
- R^2 of .1735
- Death_Funeral makes up 6.59% of the news on average

In [28]:
funeral_lm = lm(Death_Funeral*100 ~ rural_pct, data=df)
summary(funeral_lm)


Call:
lm(formula = Death_Funeral * 100 ~ rural_pct, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-14.034  -4.356  -1.791   0.526  49.324 

Coefficients:
            Estimate Std. Error t value Pr(>|t|)    
(Intercept)  2.14929    0.74351   2.891  0.00412 ** 
rural_pct    0.13683    0.01702   8.041 1.93e-14 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 8.766 on 308 degrees of freedom
Multiple R-squared:  0.1735,	Adjusted R-squared:  0.1708 
F-statistic: 64.66 on 1 and 308 DF,  p-value: 1.929e-14


## Interpretation: Democratic_Primary ~ lesscollege_pct + rural_pct + median_hh_inc 
- 15.76% of variance explained 
- pct of people with less than college degree. Increase in 1 associated with decrease in .012 in coverage of Democratic Primary, signficant 
- median_hh_inc - incredibly small, not significant 
- foreign_born_pct - also very small, not significant 

In [52]:
mean(df$Democratic_Primary)*100

[1] 0.6594489

In [56]:
dem_primary = lm(Democratic_Primary * 100 ~ lesscollege_pct + rural_pct + median_hh_inc + foreignborn_pct, data=df)
summary(dem_primary)


Call:
lm(formula = Democratic_Primary * 100 ~ lesscollege_pct + median_hh_inc + 
    foreignborn_pct, data = df)

Residuals:
    Min      1Q  Median      3Q     Max 
-1.0299 -0.2891 -0.0774  0.1884  3.6137 

Coefficients:
                  Estimate Std. Error t value Pr(>|t|)    
(Intercept)      1.380e+00  3.953e-01   3.491 0.000552 ***
lesscollege_pct -1.211e-02  3.648e-03  -3.320 0.001009 ** 
median_hh_inc    1.378e-06  2.850e-06   0.483 0.629137    
foreignborn_pct  6.647e-03  3.662e-03   1.815 0.070494 .  
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 0.4818 on 306 degrees of freedom
Multiple R-squared:  0.1657,	Adjusted R-squared:  0.1576 
F-statistic: 20.26 on 3 and 306 DF,  p-value: 5.268e-12


In [47]:
mean(df$Death_Funeral*100)

[1] 6.58987

In [9]:
df = read_csv("/shared/3/projects/benlitterer/localNews/MVP1/articleTopicsQuarter.csv")

New names:
• `` -> `...1`
Rows: 806159 Columns: 116
── Column specification ──────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
Delimiter: ","
chr  (15): article_id, sourcedomain_id, title, content, url_x, source, descr...
dbl  (99): ...1, index, docNum, Trump, Crime/Prosecution, Subscription_Servi...
lgl   (1): national
dttm  (1): date

ℹ Use `spec()` to retrieve the full column specification for this data.
ℹ Specify the column types or set `show_col_types = FALSE` to quiet this message.


In [10]:
month = month(df$date)
year = year(df$date)
df$month_year = paste(month, year, sep="_")

In [13]:
colnames(df)

[1] "...1"                               "index"                             
  [3] "docNum"                             "Trump"                             
  [5] "Crime/Prosecution"                  "Subscription_Service"              
  [7] "Media"                              "Labor/Pandemic/Immigration"        
  [9] "Isreal/Palestine/India"             "School/Universities"               
 [11] "Royal_Family/Stars"                 "NS1"                               
 [13] "Outdoor Sports"                     "Business"                          
 [15] "Data/Information/Privacy"           "Coronavirus_General"               
 [17] "States1"                            "Congress"                          
 [19] "Middle_East_Conflict"               "NS2"                               
 [21] "Community"                          "Military"                          
 [23] "Democratic_Primary"                 "NS3"                               
 [25] "NS4"                                "China"                             
 [27] "BLM/George_Floyd"                   "Sports1"                           
 [29] "Police_Activity"                    "Months"                            
 [31] "Trump/Scandals"                     "Southern/Midwestern/Eastern_States"
 [33] "Southwestern_States"                "Severe_Weather"                    
 [35] "Travel/Flights"                     "Baseball"                          
 [37] "Religion"                           "Basketball"                        
 [39] "Race"                               "Construction"                      
 [41] "Europe"                             "Political_Scandal_General"         
 [43] "2020_Election"                      "Veterans"                          
 [45] "America_as_Institution"             "Hockey"                            
 [47] "Family"                             "German_Words"                      
 [49] "Art/Books/Museums"                  "Football"                          
 [51] "Food Service"                       "NS5"                               
 [53] "Supreme_Court"                      "Covid_Illness"                     
 [55] "Sports2"                            "Climate_Fossil_Fuel"               
 [57] "Death/Funeral"                      "Covid_Economic"                    
 [59] "Vaccination"                        "Pandemic_Funding"                  
 [61] "Covid_Hospitals_Harm_Mitigation"    "Covid_Negatives/Deaths"            
 [63] "Britain"                            "NS6"                               
 [65] "Entertainment"                      "Health_Products"                   
 [67] "Science/Space"                      "Wildlife_Farms"                    
 [69] "NS7"                                "City_Government"                   
 [71] "Soccer"                             "2020_Election_Fradulence"          
 [73] "Community_Public_Service"           "article_id"                        
 [75] "sourcedomain_id"                    "date"                              
 [77] "title"                              "content"                           
 [79] "url_x"                              "fips"                              
 [81] "source"                             "description"                       
 [83] "onlinesince"                        "rank"                              
 [85] "state"                              "city"                              
 [87] "lon"                                "lat"                               
 [89] "county"                             "total_population"                  
 [91] "white_pct"                          "black_pct"                         
 [93] "hispanic_pct"                       "nonwhite_pct"                      
 [95] "foreignborn_pct"                    "female_pct"                        
 [97] "age29andunder_pct"                  "age65andolder_pct"                 
 [99] "median_hh_inc"                      "clf_unemploy_pct"                  
[1

## Model Idea: 
Wildlife and Farms % ~ Household Income + white % + rural %

### just demographics 

## Interpretation 
- White percentage: Increase in 1 percent whiteness of county associated with increase in .0115 % in topic coverage. 
- Rural percentage: Increase in 1 percent associated with .015% increase in topic. Increase in 20% percent rural = increase in ~ .3% topic coverage.  

In [14]:
# very helpful: https://m-clark.github.io/mixed-models-with-R/random_intercepts.html
# remember: ignoring clustering gives standard errors that are too small because we don't have truly 
# independant samples. Rather than n ind. samples we have something less than that. 

#na.omit behavior is default, so we only use local newspapers here 
rural_lm = lm(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct, data=df)
summary(rural_lm)


Call:
lm(formula = Wildlife_Farms * 100 ~ median_hh_inc + white_pct + 
    rural_pct, data = df)

Residuals:
   Min     1Q Median     3Q    Max 
-2.811 -1.690 -1.282 -0.819 94.919 

Coefficients:
                Estimate Std. Error t value Pr(>|t|)    
(Intercept)   -1.960e-01  6.431e-02  -3.048   0.0023 ** 
median_hh_inc  7.833e-06  9.930e-07   7.888 3.08e-15 ***
white_pct      1.147e-02  6.546e-04  17.527  < 2e-16 ***
rural_pct      1.531e-02  6.172e-04  24.801  < 2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 6.608 on 359176 degrees of freedom
  (446979 observations deleted due to missingness)
Multiple R-squared:  0.006249,	Adjusted R-squared:  0.00624 
F-statistic: 752.8 on 3 and 359176 DF,  p-value: < 2.2e-16


In [ ]:
# quick check to make sure source is available for all local 
head(df)[c("source", "national")]

In [ ]:
colnames(df)

## An attempt at some interpretation 

## adding source as random effect 

### random effects 
- On average, our estimated source intercepts are 1.132 % away from the mean in terms of topic coverage of wildlife and farming. 
- The residual variation after estimating random source intercepts is still very very large (42.870) 

### coefficients 
- an increase of 1% in white percent is associated with an increase of .0147 in increase in the proportion of content with topic of Wildlife_Farms. So 20% increase in white -> ~ .3% increase in topic. 
- ^ 20% in rural percent -> ~.14% increase in topic 
- median_hh_income has very small effect 

In [78]:
rural_source = lmer(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct + (1 | source ), data=df)
summary(rural_source)
confint(rural_source)

Warning message:
“Some predictor variables are on very different scales: consider rescaling”


Linear mixed model fit by REML ['lmerMod']
Formula: Wildlife_Farms * 100 ~ median_hh_inc + white_pct + rural_pct +  
    (1 | source)
   Data: df

REML criterion at convergence: 2370130

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.2627 -0.2682 -0.1326 -0.0888 14.3109 

Random effects:
 Groups   Name        Variance Std.Dev.
 source   (Intercept)  1.132   1.064   
 Residual             42.870   6.548   
Number of obs: 359180, groups:  source, 310

Fixed effects:
               Estimate Std. Error t value
(Intercept)   6.799e-02  3.400e-01   0.200
median_hh_inc 3.272e-06  4.776e-06   0.685
white_pct     1.471e-02  3.809e-03   3.862
rural_pct     7.021e-03  3.034e-03   2.314

Correlation of Fixed Effects:
            (Intr) mdn_h_ wht_pc
medin_hh_nc -0.698              
white_pct   -0.404 -0.318       
rural_pct   -0.324  0.597 -0.535
fit warnings:
Some predictor variables are on very different scales: consider rescaling

Computing profile confidence intervals ...



2.5 %         97.5 %      
.sig01         9.714763e-01 1.152928e+00
.sigma         6.532391e+00 6.562688e+00
(Intercept)   -5.961240e-01 7.320562e-01
median_hh_inc -6.056638e-06 1.260026e-05
white_pct      7.270035e-03 2.215146e-02
rural_pct      1.093509e-03 1.294827e-02

## Adding month fixed effect
- My understanding is that since no pooling occurs (we simply optimize each month using the data from that month), our estimates may be worse.. Other than that why can't we code as dummy variable? 
- Is it because we don't get interpretation for month effects? 

In [15]:
rural_source = lmer(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct + month_year + (1 | source ), data=df)
summary(rural_source)
#confint(rural_source)

Warning message:
“Some predictor variables are on very different scales: consider rescaling”

Correlation matrix not shown by default, as p = 24 > 12.
Use print(obj, correlation=TRUE)  or
    vcov(obj)        if you need it




Linear mixed model fit by REML ['lmerMod']
Formula: Wildlife_Farms * 100 ~ median_hh_inc + white_pct + rural_pct +  
    month_year + (1 | source)
   Data: df

REML criterion at convergence: 2369906

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.3280 -0.2639 -0.1421 -0.0846 14.3160 

Random effects:
 Groups   Name        Variance Std.Dev.
 source   (Intercept)  1.136   1.066   
 Residual             42.836   6.545   
Number of obs: 359180, groups:  source, 310

Fixed effects:
                    Estimate Std. Error t value
(Intercept)       -1.921e-01  3.440e-01  -0.558
median_hh_inc      3.248e-06  4.782e-06   0.679
white_pct          1.516e-02  3.815e-03   3.974
rural_pct          6.807e-03  3.039e-03   2.240
month_year10_2020  1.451e-01  7.000e-02   2.074
month_year10_2021  2.793e-01  7.142e-02   3.911
month_year11_2020  6.038e-02  7.157e-02   0.844
month_year11_2021  2.612e-01  7.248e-02   3.604
month_year12_2020  1.171e-01  7.143e-02   1.640
month_year12_2021  1.1

## Interpretation: 
- white: 20% increase associated with .3% topic increase 
- similar source variance
- month_year doesn't seem to vary much..? This doesn't make sense to me.. 

In [18]:
rural_source_time = lmer(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct + (1|month_year) + (1 | source ), data=df)
summary(rural_source_time)
#confint(rural_source)

Warning message:
“Some predictor variables are on very different scales: consider rescaling”


Linear mixed model fit by REML ['lmerMod']
Formula: Wildlife_Farms * 100 ~ median_hh_inc + white_pct + rural_pct +  
    (1 | month_year) + (1 | source)
   Data: df

REML criterion at convergence: 2369901

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.3238 -0.2640 -0.1416 -0.0854 14.3161 

Random effects:
 Groups     Name        Variance Std.Dev.
 source     (Intercept)  1.13518 1.0654  
 month_year (Intercept)  0.03675 0.1917  
 Residual               42.83588 6.5449  
Number of obs: 359180, groups:  source, 310; month_year, 21

Fixed effects:
               Estimate Std. Error t value
(Intercept)   6.053e-02  3.430e-01   0.176
median_hh_inc 3.250e-06  4.782e-06   0.680
white_pct     1.513e-02  3.815e-03   3.967
rural_pct     6.822e-03  3.038e-03   2.245

Correlation of Fixed Effects:
            (Intr) mdn_h_ wht_pc
medin_hh_nc -0.693              
white_pct   -0.401 -0.318       
rural_pct   -0.322  0.597 -0.535
fit warnings:
Some predictor variables are on very dif

## BLM topic ~ hh_income + white percent + rural %
- rural interpretation: a 1 percent increase in rural percentage is associated with a -.0138 change in coverage of BLM topic. 20% increase in rural -> ~ .28% increase in BLM 
- white % interpretation: a 1 percent increase in white % is associated with a .0073 change in BLM topic. So 20% increase in white -> ~.14% increase in BLM topic 
- hh income has very very small non-significant effect 

In [21]:
df = rename(df, "BLM"="BLM/George_Floyd")

In [22]:
BLM_lm = lm(BLM*100~median_hh_inc+white_pct+rural_pct, data=df)

#lmer(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct + (1|month_year) + (1 | source ), data=df)
summary(BLM_lm)


Call:
lm(formula = BLM * 100 ~ median_hh_inc + white_pct + rural_pct, 
    data = df)

Residuals:
   Min     1Q Median     3Q    Max 
-1.709 -1.483 -1.312 -0.967 95.087 

Coefficients:
                Estimate Std. Error t value Pr(>|t|)    
(Intercept)    1.186e+00  6.374e-02  18.615   <2e-16 ***
median_hh_inc -4.570e-07  9.843e-07  -0.464    0.642    
white_pct      7.299e-03  6.488e-04  11.250   <2e-16 ***
rural_pct     -1.388e-02  6.118e-04 -22.685   <2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 6.55 on 359176 degrees of freedom
  (446979 observations deleted due to missingness)
Multiple R-squared:  0.002001,	Adjusted R-squared:  0.001993 
F-statistic:   240 on 3 and 359176 DF,  p-value: < 2.2e-16


## TODO: check VIF scores 

## PROBLEM: white has negative effect alone, but positive effect with rural. Likely due to covariance between them? 

In [23]:
BLM_white = lm(BLM*100~white_pct, data=df)
summary(BLM_white)


Call:
lm(formula = BLM * 100 ~ white_pct, data = df)

Residuals:
   Min     1Q Median     3Q    Max 
-1.412 -1.311 -1.273 -1.245 95.311 

Coefficients:
             Estimate Std. Error t value Pr(>|t|)    
(Intercept)  1.425883   0.037641  37.881  < 2e-16 ***
white_pct   -0.001695   0.000525  -3.228  0.00125 ** 
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 6.556 on 359178 degrees of freedom
  (446979 observations deleted due to missingness)
Multiple R-squared:  2.901e-05,	Adjusted R-squared:  2.623e-05 
F-statistic: 10.42 on 1 and 359178 DF,  p-value: 0.001247


# Interpretation: 
- increase in 1% whiteness associated with change of of -.1% BLM_topic coverage 

In [24]:
BLM_rural = lm(BLM*100~rural_pct, data=df)
summary(BLM_rural)


Call:
lm(formula = BLM * 100 ~ rural_pct, data = df)

Residuals:
   Min     1Q Median     3Q    Max 
-1.576 -1.514 -1.283 -0.997 95.152 

Coefficients:
              Estimate Std. Error t value Pr(>|t|)    
(Intercept)  1.5763786  0.0155269  101.53   <2e-16 ***
rural_pct   -0.0105129  0.0004346  -24.19   <2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 6.551 on 359178 degrees of freedom
  (446979 observations deleted due to missingness)
Multiple R-squared:  0.001627,	Adjusted R-squared:  0.001624 
F-statistic: 585.2 on 1 and 359178 DF,  p-value: < 2.2e-16


In [25]:
BLM_lm = lm(BLM*100~median_hh_inc+white_pct+rural_pct, data=df)

#lmer(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct + (1|month_year) + (1 | source ), data=df)
summary(BLM_lm)


Call:
lm(formula = BLM * 100 ~ median_hh_inc + white_pct + rural_pct, 
    data = df)

Residuals:
   Min     1Q Median     3Q    Max 
-1.709 -1.483 -1.312 -0.967 95.087 

Coefficients:
                Estimate Std. Error t value Pr(>|t|)    
(Intercept)    1.186e+00  6.374e-02  18.615   <2e-16 ***
median_hh_inc -4.570e-07  9.843e-07  -0.464    0.642    
white_pct      7.299e-03  6.488e-04  11.250   <2e-16 ***
rural_pct     -1.388e-02  6.118e-04 -22.685   <2e-16 ***
---
Signif. codes:  0 ‘***’ 0.001 ‘**’ 0.01 ‘*’ 0.05 ‘.’ 0.1 ‘ ’ 1

Residual standard error: 6.55 on 359176 degrees of freedom
  (446979 observations deleted due to missingness)
Multiple R-squared:  0.002001,	Adjusted R-squared:  0.001993 
F-statistic:   240 on 3 and 359176 DF,  p-value: < 2.2e-16


## Interpretation: 
- source is 1.5% away from mean on average

## BLM topic % ~ demographics + source random effect 

In [27]:
BLM_lm = lmer(BLM*100~median_hh_inc+white_pct+rural_pct+(1|source), data=df)

#lmer(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct + (1|month_year) + (1 | source ), data=df)
summary(BLM_lm)

Warning message:
“Some predictor variables are on very different scales: consider rescaling”


Linear mixed model fit by REML ['lmerMod']
Formula: BLM * 100 ~ median_hh_inc + white_pct + rural_pct + (1 | source)
   Data: df

REML criterion at convergence: 2362419

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-0.9151 -0.2295 -0.1566 -0.0569 14.5569 

Random effects:
 Groups   Name        Variance Std.Dev.
 source   (Intercept)  1.25    1.118   
 Residual             41.96    6.477   
Number of obs: 359180, groups:  source, 310

Fixed effects:
                Estimate Std. Error t value
(Intercept)    1.353e+00  3.558e-01   3.804
median_hh_inc -1.160e-05  4.995e-06  -2.323
white_pct      1.276e-02  3.988e-03   3.201
rural_pct     -1.500e-02  3.174e-03  -4.726

Correlation of Fixed Effects:
            (Intr) mdn_h_ wht_pc
medin_hh_nc -0.698              
white_pct   -0.404 -0.318       
rural_pct   -0.324  0.598 -0.535
fit warnings:
Some predictor variables are on very different scales: consider rescaling

## Interpretation: 
- source and month_year each vary from mean topic coverage by ~1%. White and rural have positive, negative effects 
of ~.1% increase/decrease in topic coverage for 1% increase in white/rural 

In [28]:
BLM_lm = lmer(BLM*100~median_hh_inc+white_pct+rural_pct+(1|source)+(1|month_year), data=df)

#lmer(Wildlife_Farms*100~median_hh_inc+white_pct+rural_pct + (1|month_year) + (1 | source ), data=df)
summary(BLM_lm)

Warning message:
“Some predictor variables are on very different scales: consider rescaling”


Linear mixed model fit by REML ['lmerMod']
Formula: BLM * 100 ~ median_hh_inc + white_pct + rural_pct + (1 | source) +  
    (1 | month_year)
   Data: df

REML criterion at convergence: 2351935

Scaled residuals: 
    Min      1Q  Median      3Q     Max 
-1.5926 -0.2442 -0.1245 -0.0148 14.8332 

Random effects:
 Groups     Name        Variance Std.Dev.
 source     (Intercept)  1.021   1.011   
 month_year (Intercept)  1.169   1.081   
 Residual               40.740   6.383   
Number of obs: 359180, groups:  source, 310; month_year, 21

Fixed effects:
                Estimate Std. Error t value
(Intercept)    1.300e+00  4.004e-01   3.247
median_hh_inc -1.062e-05  4.546e-06  -2.336
white_pct      1.035e-02  3.625e-03   2.856
rural_pct     -1.322e-02  2.889e-03  -4.576

Correlation of Fixed Effects:
            (Intr) mdn_h_ wht_pc
medin_hh_nc -0.564              
white_pct   -0.326 -0.318       
rural_pct   -0.262  0.597 -0.535
fit warnings:
Some predictor variables are on very different